satellite/repair/repairer/peer.go

```diff
@@ -0,0 +1,169 @@
+// Copyright (C) 2019 Storj Labs, Inc.
```

Comment:
i think we want a similar structure for things. For example, the apiProcess is located at `satellite/api.go` so i think we may want the repair worker process in a similar place? Thoughts?